storages/backends/gcloud.py

```diff
@@ -56,6 +57,8 @@ def _get_file(self):
                 self._is_dirty = False
                 self.blob.download_to_file(self._file)
                 self._file.seek(0)
+            if self._storage.gzip and self.blob.content_encoding == 'gzip':
+                self._file = self._compress_file()
```

Comment:
Doesn't this actually decompress rather than compress? If the content_encoding is gzip already and we are getting the file and all.

This might be buggy actually, not sure. Why would `self._storage.gzip` matter in this case.

Reply:
Good catch! As you can see in the CompressFileMixin the `_compress_file` method is actually decompressing the file. I just gave it the wrong name.